lib/galaxy/webapps/galaxy/api/users.py

```diff
@@ -730,6 +731,19 @@ def send_activation_email(
         if not self.service.user_manager.send_activation_email(trans, user.email, user.username):
             raise exceptions.MessageException("Unable to send activation email.")
 
+    @router.get(
+        "/api/users/{user_id}/roles",
+        name="get user roles",
+        description="Return a collection of roles associated with this user. Only admins can see user roles.",
```

Comment:
We should avoid the name `collection` here if possible to avoid confusion with "Galaxy's collections".

Reply:
Replaced with "list". I think list is generic enough in this context, but we can reword if needed.